README.md

```diff
@@ -71,6 +71,48 @@ Then use the `-l` option to specify the stage/environment
 tape ansible deploy -l staging
 ```
 
+### Docker
+
+** Install **
+Follow instructions above but when intalling add the `-d` flag to indicate a
+docker project.
+
+```sh
+tape installer install -d
+```
+
+** Configure **
+Tape gives you three docker specific files.  They contain the defaults for
+running a Node.js app.
+
+* Dockerfile: This will be coppied to the correct place on the server.  It
+  will work as-is, but probably should be editted. Can be coppied to project root
+  to run docker in development
+* docker-compose.yml:  To be used in development. To use, copy to the root
+  directory
+* docker-compose.production.yml: This will be coppied to the correct place on
+  the server. Should be eddited to set up correct services (db, redis, etc) and
+  to correctly mount volumes and set environment variables
+
+** Test Docker Setup **
+
+[Install Docker](https://docs.docker.com/mac/)
+
+```sh
+cp taperole/Dockerfile .
+cp taperole/docker-compose.yml
+
+docker-compose up
```

Comment:
Lets write a docker runner that encapsulates all this.

Runn
```
take docker everything
```
:top: that command will cd into the taperole directory and run the docker-compose up command for you making this process a lot easier. It will then be simpler to expand that "docker runner" out to perform other functionality.

Essentially, this will mean that when you use tape you can etiher

A) Use ansible
B) User docker

Then eventually we will deprecate the concept of separate runners entirely in favor of a pure docker setup!